src/structures/Guild.js

```diff
@@ -926,6 +926,7 @@ class Guild extends Base {
 
   /**
    * The data for editing a guild.
+   * <info>Only the `COMMUNITY` feature can be modified</info>
```

Comment:
Do we want to document this? It'd mean we have to update it every time Discord adds new ones (also pretty sure it's NOT the only one)

Reply:
every other feature is a privilege you can't self-grant or disable so far